modules/GoogleDialer/Android.mk

```diff
@@ -9,3 +9,14 @@ GAPPS_LOCAL_OVERRIDES_MIN_VARIANT :=
 GAPPS_LOCAL_OVERRIDES_PACKAGES := Dialer
 
 include $(BUILD_GAPPS_PREBUILT_APK)
+include $(CLEAR_VARS)
+include $(GAPPS_CLEAR_VARS)
+
+LOCAL_MODULE := GoogleDialer_dialer_experience.xml
+LOCAL_MODULE_STEM := dialer_experience.xml
+LOCAL_MODULE_TAGS := optional
+LOCAL_MODULE_CLASS := ETC
+LOCAL_SRC_FILES := $(GAPPS_SOURCES_PATH)/all/etc/sysconfig/dialer_experience.xml
+LOCAL_MODULE_PATH := $(TARGET_OUT_ETC)/sysconfig
+
+include $(BUILD_PREBUILT)
```

Comment:
is there an advantage to having a module defined for GoogleDialer_dialer_experience.xml?

since it is just a single file that needs to be copied (or not copied), why not just use PRODUCT_COPY_FILES?

Reply:
Just that if someone runs `mmm vendor/opengapps/build/modules/GoogleDialer` the file gets included. It also makes the dependency more explicit.

Reply:
to make the dependency really explicit, you could do this:

```diff
$ git diff
diff --git a/modules/GoogleDialer/Android.mk b/modules/GoogleDialer/Android.mk
index c49aa3a..b772773 100644
--- a/modules/GoogleDialer/Android.mk
+++ b/modules/GoogleDialer/Android.mk
@@ -8,6 +8,9 @@ LOCAL_PRIVILEGED_MODULE := true
 GAPPS_LOCAL_OVERRIDES_MIN_VARIANT :=
 GAPPS_LOCAL_OVERRIDES_PACKAGES := Dialer
 
+# express dependency on xml file
+GoogleDialer: GoogleDialer_dialer_experience.xml
+
 include $(BUILD_GAPPS_PREBUILT_APK)
 
 include $(CLEAR_VARS)
```
This way the xml file will get copied even when someone runs `m GoogleDialer`.

Reply:
after looking at some of the other makefiles, I see you can do the same thing using `LOCAL_REQUIRED_MODULES`.  That is probably the preferred way.

```diff
$ git diff
diff --git a/modules/GoogleDialer/Android.mk b/modules/GoogleDialer/Android.mk
index c49aa3a..d75a84c 100644
--- a/modules/GoogleDialer/Android.mk
+++ b/modules/GoogleDialer/Android.mk
@@ -8,6 +8,8 @@ LOCAL_PRIVILEGED_MODULE := true
 GAPPS_LOCAL_OVERRIDES_MIN_VARIANT :=
 GAPPS_LOCAL_OVERRIDES_PACKAGES := Dialer
 
+LOCAL_REQUIRED_MODULES := GoogleDialer_dialer_experience.xml
+
 include $(BUILD_GAPPS_PREBUILT_APK)
 
 include $(CLEAR_VARS)
```